test/integration2/schema/misc.spec.js

```diff
@@ -1576,6 +1576,21 @@ describe('Schema (misc)', () => {
           knex.schema.hasTable('').then((resp) => {
             expect(resp).to.equal(false);
           }));
+
+        it('should not parse table name if wrapIdentifier is not specified', () => {
+          knex.schema.hasTable('testTableTwo').then((resp) => {
+            expect(resp).to.equal(false);
+          });
+        });
+
+        it('should parse table name if wrapIdentifier is specified', () => {
+          knex.client.config.wrapIdentifier = (value, origImpl, queryContext) =>
+            origImpl(_.snakeCase(value));
+
+          knex.schema.hasTable('testTableTwo').then((resp) => {
+            expect(resp).to.equal(true);
+          });
```

Comment:
Since these methods return promises we need to let the test know that its result is async by returning the promise from it or using async/await (as it is done in other tests of this file):

```suggestion
        it('should not parse table name if wrapIdentifier is not specified', async () => {
          const resp = await knex.schema.hasTable('testTableTwo')
          expect(resp).to.equal(false);
        });

        it('should parse table name if wrapIdentifier is specified', async () => {
          knex.client.config.wrapIdentifier = (value, origImpl, queryContext) =>
            origImpl(_.snakeCase(value));

          const resp = await knex.schema.hasTable('testTableTwo')
          expect(resp).to.equal(true);
```

Reply:
Hey, thanks for your suggestion!
I thought of using async/await, but I saw that the other test cases of `hasTable` were using `.then`, so I based myself on that.
https://github.com/knex/knex/blob/master/test/integration2/schema/misc.spec.js#L1564

Reply:
Thanks for your suggestion! I haven't noticed before :smile: